decidim-accountability/config/locales/en.yml

```diff
@@ -159,8 +159,8 @@ en:
             progress: Progress
         timeline_entry:
           fields:
-            description: Description
```

Comment:
I think I skipped this yesterday, but do we really need to remove the `description` key here?

Reply:
This translation is used in the Admin side, in the timeline_entries#index view, I changed to show the `title` instead of the `decription`:

https://github.com/decidim/decidim/blob/50a5d942107431c975be98c1ac0ab26760c61811/decidim-accountability/app/views/decidim/accountability/admin/timeline_entries/index.html.erb#L16

Reply:
Oh I see, that seems right, sorry.

I think I've messed up, I thought that these were the fields that are used on the form. I see that in this case it's missing (`activemodel.attributes.timeline_entry.title` isn't there at line 23. Can you check that please?

Reply:
What I'm saying is that the source string "Title" is missing in `en.yml` so the translation will not be added to this attribute:

```diff
diff --git a/decidim-accountability/config/locales/en.yml b/decidim-accountability/config/locales/en.yml
index 1688c73b3b..5207004161 100644
--- a/decidim-accountability/config/locales/en.yml
+++ b/decidim-accountability/config/locales/en.yml
@@ -22,6 +22,7 @@ en:
       timeline_entry:
         description: Description
         entry_date: Date
+        title: Title
     models:
       decidim/accountability/proposal_linked_event: Proposal included in a result
       decidim/accountability/result_progress_updated_event: Result progress updated
```

If it's not added on English there isn't any problem but in other locales this will not get translated: 

![image](https://user-images.githubusercontent.com/717367/165898348-99ca833b-b6c5-43df-a96c-5940f5f640a1.png)

Reply:
I think @andreslucena is suggesting we need to add `title` also here to make the field translatable:
https://github.com/decidim/decidim/blob/50a5d942107431c975be98c1ac0ab26760c61811/decidim-accountability/config/locales/en.yml#L22-L24

Reply:
Ok, got it. Thanks! 😄